bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java

```diff
@@ -133,11 +134,17 @@ public static void addProperties(StyleToken styleToken, String properties) {
      * @param styleToken the {@link StyleToken}
      * @return a valid CSS representation
      */
+    @Nullable
     public static String toCss(StyleToken styleToken) {
-        if (Objects.isNull(styleToken)) {
-            return null;
+        String css = null;
+        if (Objects.nonNull(styleToken)) {
+            if (styleToken.isMediaQuery()) {
```

Comment:
Why do we make a difference here between media query and other selectors? Could we use the same pattern (with the spaces) for all?

Reply:
@Buuhuu as discussed offline I updated the PR to unify the behaviour for all css selectors and adapt the formatting to the behaviour in the mjml project.